src/plone/restapi/tests/test_services_vocabularies.py

```diff
@@ -61,14 +61,60 @@ def test_get_vocabulary(self):
         response = response.json()
         self.assertEqual(
             response,
-            {u'@id': u'http://localhost:55001/plone/@vocabularies/''plone.restapi.tests.test_vocabulary',  # noqa
-             u'terms': [
-                 {u'@id': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary/token1',  # noqa
-                  u'title': u'Title 1',
-                  u'token': u'token1'},
-                 {u'@id': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary/token2',  # noqa
-                  u'title': u'Title 2',
-                  u'token': u'token2'}]})
+            {
+                u'@id': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary',  # noqa
+                u'terms': [
+                    {u'title': u'Title 1', u'token': u'token1'},
+                    {u'title': u'Title 2', u'token': u'token2'},
+                ],
+                u'terms_total': 2,
+            }
+        )
+
+    def test_get_vocabulary_batched(self):
+        response = self.api_session.get(
+            '/@vocabularies/plone.restapi.tests.test_vocabulary?b_size=1'
+        )
+
+        self.assertEqual(200, response.status_code)
+        response = response.json()
+        self.assertEqual(
+            response,
+            {
+                u'@id': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary',  # noqa
+                u'batching': {
+                    u'@id': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary?b_size=1',  # noqa
+                    u'first': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary?b_start=0&b_size=1',  # noqa
+                    u'last': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary?b_start=1&b_size=1',  # noqa
+                    u'next': u'http://localhost:55001/plone/@vocabularies/plone.restapi.tests.test_vocabulary?b_start=1&b_size=1',  # noqa
+                },
+                u'terms': [
```

Comment:
@lukasgraf I can rename it to `items`. I was thinking that it should be named something different since the entries in the list are not content items, but your argument about being able to reuse client-side batching support convinced me.

Reply:
@davisagli there's some history behind `items`: Back when Hydra was still on the table, it started as `member`, but [`__parent__` and `__children__` properties were also being discussed](https://github.com/plone/plone.restapi/issues/5) :trollface: So, as a general term for "sequence of things" we could do much worse than `items` 😉

Reply:
@lukasgraf it also matches the terminology of JSON Schema